Model/simulation.py

```diff
@@ -274,23 +273,24 @@ def top_up_sequence(self, block_height):
                     f"  Allocation transition FAILED for vault {vault.id}: {str(e)}"
                 )
 
-    def spend_sequence(self, block_height):
-        logging.info(f"Spend sequence at block {block_height}")
+    def spend(self, block_height):
         if len(self.wt.list_available_vaults()) == 0:
             raise NoVaultToSpend
 
         vault_id = random.choice(self.wt.list_available_vaults()).id
         # Spend transition
-        logging.info(f"  Spend transition at block {block_height}")
+        logging.info(
+            f"  Spend transition with vault {vault_id} at block {block_height}"
+        )
         self.wt.spend(vault_id, block_height)
 
         # snapshot coin pool after spend attempt
         if self.with_coin_pool:
             amounts = [coin.amount for coin in self.wt.list_coins()]
             self.pool_after_spend.append([block_height, amounts])
 
-    def cancel_sequence(self, block_height):
-        logging.info(f"Cancel sequence at block {block_height}")
+    def cancel(self, block_height):
```

Comment:
Why this renaming?

Reply:
to be consistent with the paper and our refactoring of the state transitions. Now that there's only `broadcast_cancel` transition in the "sequence" it's not really a sequence, it's just a cancel.